lib/derivativo/conversion/office_helpers.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module Derivativo
+  module Conversion
+    module OfficeHelpers
+      PDF_HIGHER_COMPRESSION_CONVERSION_THRESHOLD = 30.megabytes
+
+      # Converts an input office output audiovisual file
+      def self.office_convert_to_pdf(src_file_path:, dst_file_path:)
+        office_convert_to_pdf_impl(src_file_path: src_file_path, dst_file_path: dst_file_path, soffice_binary_path: soffice_binary_path_from_config_or_path)
+
+        # If the generated file size is less than or equal to our desired size, then we're done.
+        generated_file_size = File.size(dst_file_path)
```

Comment:
If office_convert_to_pdf_impl fails to convert, File.size will throw/raise Errno::ENOENT

Reply:
Good point.  I'll raise an intentional Hyacinth::Exceptions::ConversionError in the office_convert_to_pdf_impl method and catch it elsewhere instead.